JesusFilm/core
# fix: add new icons for rtl use

# Description

These are the front-end changes needed for icon RTL. We're adding two new icons to the icon list: ArrowBackRounded, and ChevronLeftRounded.

Depends on backend changes #974 

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29688713/todos/5413250221)

# How should this PR be QA Tested?

On Admin

1. Visit any published journey (or non-published but requires to be published)
2. Edit the cards
3. Add a button (or edit a button if one exists)
4. Add the icons - ArrowBackRounded and ChevronLeftRounded
5. After adding, preview the journey

- [x] it should enable users to add the ArrowBackRounded icon or Arrow Left
- [x] it should enable users to add the ChevronLeftRounded icon
- [x] it should render newly added icons ArrowBackRounded and ChevronLeftRounded


# Checklist
